src/containers/components/Navbar/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import "./style.css"
+export default function Navbar() {
+  return (
+    <div>
+        <ul className="nav bg-dark">
+  <li className="nav-item">
+    <a className="nav-link active" href='/' aria-current="page"><img alt='logoITviec' style={{width:"50%"}} src='https://itviec.com/assets/logo-itviec-4492a2f2577a15a0a1d55444c21c0fa55810822b3b189fc689b450fb62ce0b5b.png'/></a>
+  </li>
+  <li className="nav-item">
+    <a className="nav-link" href='Alljobs'>All jobs</a>
```

Comment:
you can use `href='/#'` to prevent reload page

Reply:
i have changed it

Reply:
apply with other link elements

Reply:
i've changed